src/commands/account/create_account/fund_myself_create_account/sign_as/mod.rs

```diff
@@ -0,0 +1,446 @@
+                            ..
+                        },
+                    ),
+                )) => {
+                    break false;
+                }
+                Err(near_jsonrpc_client::errors::JsonRpcError::ServerError(_)) => {
+                    println!(
+                        "Unable to verify the existence of account <{}> on network <{}>",
+                        account_id, network.1.network_name
+                    );
+                    break false;
+                }
+            }
+        };
+    }
+    Ok(None)
+}
+
+async fn is_signer_account_id(
+    network_config: &crate::config::NetworkConfig,
+    account_id: &near_primitives::types::AccountId,
+) -> color_eyre::eyre::Result<bool> {
+    for retries_left in (0..5).rev() {
+        match crate::common::get_account_state(
+            network_config.clone(),
+            account_id.clone(),
+            near_primitives::types::Finality::Final.into(),
+        )
+        .await
+        {
+            Ok(optional_account_view) => {
+                if optional_account_view.is_none() {
+                    return color_eyre::eyre::Result::Err(color_eyre::eyre::eyre!(
+                        "\nSigner account <{}> does not yet exist on the network <{}>.",
+                        account_id,
+                        network_config.network_name
+                    ));
+                } else {
+                    return Ok(true);
+                }
+            }
+            Err(near_jsonrpc_client::errors::JsonRpcError::TransportError(err)) => {
+                println!("Transport error request. If you want to exit the program, press ^C.\nThe next try to send this request is happening right now. Please wait ...");
+                if retries_left == 0 {
+                    return color_eyre::eyre::Result::Err(color_eyre::eyre::eyre!("{err}"));
+                }
+                tokio::time::sleep(std::time::Duration::from_millis(1000)).await;
+            }
+            Err(near_jsonrpc_client::errors::JsonRpcError::ServerError(
+                near_jsonrpc_client::errors::JsonRpcServerError::HandlerError(
+                    near_jsonrpc_primitives::types::query::RpcQueryError::UnknownAccount {
+                        requested_account_id,
+                        ..
+                    },
+                ),
+            )) => {
+                return color_eyre::eyre::Result::Err(color_eyre::eyre::eyre!(
+                    "Signer account {requested_account_id} does not exist now."
+                ));
+            }
+            Err(near_jsonrpc_client::errors::JsonRpcError::ServerError(err)) => {
+                return color_eyre::eyre::Result::Err(color_eyre::eyre::eyre!(err.to_string()))
+            }
+        }
+    }
+    Ok(true)
+}
+
+async fn is_new_account_id(
+    network_config: &crate::config::NetworkConfig,
+    account_id: &near_primitives::types::AccountId,
+) -> color_eyre::eyre::Result<bool> {
```

Comment:
Similar to the `validate_signer_account_id` comment above:

```suggestion
async fn validate_new_account_id(
    network_config: &crate::config::NetworkConfig,
    account_id: &near_primitives::types::AccountId,
) -> color_eyre::eyre::Result<()> {
```